code/style.css

```diff
@@ -1,4 +1,150 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  color: white;
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+}
+
+.header {
+  background-color: #23232e;
+}
+
+.mobile-tablet-view {
+  display: flex;
+  justify-content: space-between;
+}
+
+.navigation-container {
+  display: none;
+}
+
+.logo {
+  width: 50px;
+  padding: 10px
+}
+
+h1 {
+  margin-left: 15px;
+}
+
+.navigation-item {
+  padding: 30px 20px;
+}
+
+a {
+  text-decoration: none;
+  color:cornflowerblue;
+}
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin: 24px 24px 24px 0;
+}
+
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: #fff;
+}
+
+.big-news {
+  background-color: #6b7296;
+  margin-bottom: 30px;
+}
+ 
+.main-image {
+  width: 100%;
+  object-fit: cover;
+}
+
+h1 {
+  color: #b6c5e6;
+}
+
+h2 {
+  text-align: center;
+}
+
+.article {
+  padding: 20px 25px;
+  text-align: justify;
+}
+
+.other-news {
+  color: black;
+}
+
+.news-card {
+  padding: 30px;
+  margin: 20px;
+  background-color: white;
+  border: 1px solid black;
+  border-radius: 5px;
+  box-shadow: 7px 7px gray;
+  text-align: center;
+  transition: transform .3s;
+}
+
+.news-card:hover {
+  transform: scale(1.05);
+}
+
+.news-card img {
+  width: 100%;
+  object-fit: cover;
+}
+
+footer {
+  background-color: #23232e;
+  padding: 20px;
+}
+
+@media (max-width: 767px) {
+  h1 {
+    display: none;
+  }
+}
```

Comment:
If you would like to go with the mobile first approach, you could have this styling inline together with the rest on row 25, and then in the tablet media query - set it to `display: block;`